src/redux/actions/missions.js

```diff
@@ -1,12 +1,16 @@
 import { FETCH_ALL_MISSIONS } from './actionTypes';
 import * as api from '../../api/api';
 
-// API action creators
 const getMissions = () => async (dispatch) => {
   try {
-    const data = await api.fetchData(); // TODO: Change 'fetchData' to 'fetchMissions'
+    const data = await api.fetchMissions();
+    const filteredObject = data.map((obj) => {
+      const { mission_id: missionId, mission_name: missionName, description } = obj;
+      return { mission_id: missionId, mission_name: missionName, description };
+    });
+    console.log(filteredObject);
 
```

Comment:
# Optional Changes
 Please remember to remove the console.log in your missions.js.